### DISCUSSION TITLE
device id

### DISCUSSION THREAD

**Discussion by strangecarr** [gilbarbara/react-spotify-web-playback#115] [General]
Hi I'm trying to integrate the web player into my app (a music events listing app) using this package. I've authenticated with the correct scopes, but when I click play in my app I receive:
"Spotify Web Player: No player"

I checked dev tools/network to see the api calls and noticed:
Request URL: https://api.spotify.com/v1/me/player/play?device_id=

There is no device id in the api call - I'd expect this to be populated. Any advice on what is going wrong?

**Comment by gilbarbara** [comment]
Hey @strangecarr 

Please set up a codesandbox or similar with a minimal reproducible example.
